.github/containers/Dockerfile

```diff
@@ -96,7 +96,7 @@ RUN echo 'eval "$(pyenv init -)"' >>${HOME}/.bashrc && \
     pyenv update
 
 # Install Python
-ARG PYTHON_VERSIONS="3.11 3.10 3.9 3.8 3.7 3.12 2.7 pypy2.7-7.3.12 pypy3.8-7.3.11"
+ARG PYTHON_VERSIONS="3.11 3.10 3.9 3.8 3.7 3.12 2.7 pypy2.7-7.3.12 pypy3.10-7.3.15 pypy3.8-7.3.11"
```

Comment:
I think we can just drop pypy3.8 here now can't we?

Reply:
I think there might be at least 1 framework that doesn't support pypy3.8, I have 3 left that haven't passed yet with 3.10. Will take another look tomorrow to see if I can get them all to pass and I'll drop 3.8 if that works.

Reply:
I think I got them all running now, I dropped 3.8 completely